release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -470,6 +470,13 @@ Red{nbsp}Hat support exists for using the Kubernetes NMState Operator on {azure-
 
 For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator_k8s-nmstate-operator[About the Kubernetes NMState Operator].
 
+[id="ocp-4-17-networking-nmstate-operator-metrics_{context}"]
+==== View metrics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using  the *Administrator* and *Developer* perspectives.
```

Comment:
```suggestion
The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using the *Administrator* and *Developer* perspectives.
```
Sorry, my suggested change too had an extra space, which needs to be removed.

Reply:
Thanks. Updated!